app/script/view_model/content/TitleBarViewModel.js

```diff
@@ -109,6 +114,22 @@ z.viewModel.content.TitleBarViewModel = class TitleBarViewModel {
     this.showDetails();
   }
 
+  onMousedown() {
+    this.isDragged = this.isMacDesktop;
+  }
+
+  onMousemove() {
+    this.isMoved = this.isDragged;
+  }
+
+  onMouseup() {
+    if (this.isMoved && this.isDragged) {
+      this.preventPanelOpen = true;
+    }
```

Comment:
couldn't that be: `this.preventPanelOpen = this.isMoved && this.isDragged;` instead of the if statement?

Reply:
```true```